src/main/java/adventOfCode/Day1/Day1/spacecraft.java

```diff
@@ -0,0 +1,56 @@
+package adventOfCode.Day1.Day1;
+
+public class spacecraft {
+
+    public static int getFuelRequirementsForOneModule(int moduleMass) {
+        int fuelRequirementsForOneModule = moduleMass / 3 - 2;
```

Comment:
`moduleMass / 3 - 2;`- repeats itself in multiple places- could be moved into a function.